packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -459,6 +442,114 @@ export class TablePageObject<T extends TableRecord> {
         }
     }
 
+    /**
+     * @param columnIndex The index of the column to the left of a divider being dragged. Thus, this
+     * can not be given a value representing the last visible column index.
+     * @param deltas The series of mouse movements in the x-direction while sizing a column.
+     */
+    public dragSizeColumnByRightDivider(
+        columnIndex: number,
+        deltas: number[]
+    ): void {
+        const divider = this.getColumnRightDivider(columnIndex);
+        if (divider === null) {
+            throw new Error(
+                'The provided column index has no right divider associated with it.'
+            );
+        }
+        divider.setAttribute('active', 'true');
+        const dividerRect = divider.getBoundingClientRect();
+        let currentMouseX = (dividerRect.x + dividerRect.width) / 2;
+        const mouseDownEvent = new MouseEvent('mousedown', {
+            clientX: currentMouseX,
+            clientY: (dividerRect.y + dividerRect.height) / 2
+        });
+        divider.dispatchEvent(mouseDownEvent);
+
+        for (const delta of deltas) {
+            currentMouseX += delta;
+            const mouseMoveEvent = new MouseEvent('mousemove', {
+                clientX: currentMouseX
+            });
+            document.dispatchEvent(mouseMoveEvent);
+        }
+
+        const mouseUpEvent = new MouseEvent('mouseup');
+        document.dispatchEvent(mouseUpEvent);
+    }
+
+    /**
+     * @param columnIndex The index of the column to the right of a divider being dragged. Thus, this
+     * value must be greater than 0 and less than the total number of visible columns.
+     * @param deltas The series of mouse movements in the x-direction while sizing a column.
+     */
+    public dragSizeColumnByLeftDivider(
+        columnIndex: number,
+        deltas: number[]
+    ): void {
+        const divider = this.getColumnLeftDivider(columnIndex);
+        if (divider === null) {
```

Comment:
Any reason not to simplify this condition?
```suggestion
        if (!divider) {
```